src/obstacle_avoidance/launch/my_node.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <node name="obs_avoid_node" pkg="obstacle_avoidance" type="obs_avoid_node" output="screen">
+        <param name="character" value="?" />
```

Comment:
We can take this `param` out now, seeing as your node doesn't need to take any (I think it's left over from the sample package.